### PR TITLE
Fix issue #46

### PR DESCRIPTION
**Pull request by chriscz**

### CODE REVIEW

**Comment by cjw296** [comment]
Obviously, all the tests need to pass. I also don't expect to see any drop in coverage, although I suspect that's due to the test failures.

File: testfixtures/tempdirectory.py [hunk 1]
```diff
@@ -94,6 +94,10 @@ def actual(self,
                path=None, recursive=False, files_only=False, followlinks=False):
         path = self._join(path) if path else self.path
 
+        # strip any ending '/' in path
+        if path.endswith(os.sep) or path.endswith('/'):
+            path = path[:-1]
```
Comment (cjw296) [inline testfixtures/tempdirectory.py]: you never heard of `rstrip()`? ;-)

File: testfixtures/tests/test_tempdir.py [hunk 2]
```diff
@@ -110,3 +110,15 @@ def test_method(d):
...
+    def test_path_end_seperator(self, d):    
+        d.write('source/foo/bar.txt', '')
+
+        d.compare(path='source/', expected=['foo/', 'foo/bar.txt']) 
+
+    @tempdir()
+    def test_path_not_end_seperator(self, d):    
+        d.write('source/foo/bar.txt', '')
+
+        d.compare(path='source', expected=['foo/', 'foo/bar.txt'])
```
Comment (cjw296) [inline testfixtures/tests/test_tempdir.py]: Isn't this case already covered elsewhere?

File: testfixtures/tempdirectory.py (hunk 1 above)
Comment (chriscz) [inline testfixtures/tempdirectory.py]: For some reason that came to me before `rstrip`, will update :)

**Comment by chriscz** [comment]
I am trying to see why the coverage dropped, since running the tests after the fix shows that all tests pass on my machine.

File: testfixtures/tests/test_tempdir.py (hunk 2 above)
Comment (chriscz) [inline testfixtures/tests/test_tempdir.py]: Are you referring to `test_path_end_seperator` or both test? 
Would you prefer I merge the test into an existing test function?

**Pull request closed without merging**